tools/dockerfile/interoptest/grpc_interop_java/build_interop.sh

```diff
@@ -16,16 +16,24 @@
 # Builds Java interop server and client in a base image.
 set -e
 
-mkdir -p /var/local/git
-git clone --recursive --depth 1 /var/local/jenkins/grpc-java /var/local/git/grpc-java
+cp -r /var/local/jenkins/grpc-java /tmp/grpc-java
```

Comment:
FTR this actually makes java interop build behave differently from how other languages' build work  when running interop tests manually - cloning from the outside repo will only bring committed change, while as cp -r will bring even uncommitted changes. 
(that's probably fine for Java, but I was worried that for C/C++ it could actually potentially break the build).

Reply:
Just saw the other comment

Reply:
Also FTR, the git dependency hid a dependency added by protobuf, because git brought in that dependency. https://github.com/protocolbuffers/protobuf/issues/5875 . After my changes the container notices that libatomic was incorrectly added as a dependency.